setup.py

```diff
@@ -80,6 +86,103 @@ class Components:
 else:
     from setuptools.command.build_ext import build_ext
 
+music_home = ""
+if Components.MUSIC:
```

Comment:
@alexsavulescu Does this section get removed? I.e  installation of MUSIC managed elsewhere?